modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/io/FileSystem.scala

```diff
@@ -40,18 +43,29 @@ object FileSystem {
       path <- Stream.fromIterator(paths.iterator().asScala)
     } yield path
 
-  def readJson[F[_]: Sync](path: Path): EitherT[F, String, Json] =
+  def readJson[F[_]: Sync, A: Decoder](path: Path, fallbacks: TSConfig => TSConfig): EitherT[F, String, A] =
     Sync[F]
-      .delay[String](Files.readString(path))
+      .delay(SSource.fromFile(path.toFile).mkString)
       .attemptT
-      .leftMap(e => show"Error reading ${path.toAbsolutePath.toString} JSON file from filesystem: ${e.getMessage}")
-      .subflatMap(str => parse(str).leftMap(e => show"Cannot parse JSON in ${path.toAbsolutePath.toString}: ${e.getMessage()}"))
+      .leftMap(e => s"Cannot read file: ${e.getMessage}")
```

Comment:
```suggestion
      .leftMap(e => s"Cannot read file $path: ${e.getMessage}")
```

Reply:
This was actually deliberate.  I found that `e.getMessage` already contains the path name.  And as this is a very user-facing error message (e.g. when a developer is trying to get enrich working for the first time) I thought it was important to have a plain error message that is easy to read.  So in my opinion it's more helpful to omit the extra `$path` from the message.

Reply:
Agreed! As long as there is the path we're good!